Sources/Stagehand/AnimationInstance/Renderer.swift

```diff
@@ -70,7 +70,12 @@ internal final class Renderer<ElementType: AnyObject>: AnyRenderer {
             return
         }
 
+        CATransaction.begin()
+        CATransaction.setDisableActions(true)
```

Comment:
Ahhh this bad boy again!

Reply:
The aka "Core Animation please stop doing hard to debug magic" line